src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -206,9 +214,12 @@ export const TransactionDetailsSummaryItem: FC<TransactionDetailsSummaryItemProp
         value={shippingDisplayAmount()}
         data-test="shippingDisplayAmount"
       />
-
       <Entry
-        label="Tax*"
+        label={
+          <>
+            Tax <Sup>†</Sup>
```

Comment:
I think we still dont want to update to the dagger on regular old orders (like pickup of partner flat fee shipping) So we might need a few more conditionals here

Reply:
Im going to push some commits here!